src/include/daos_errno.h

```diff
@@ -113,7 +113,7 @@ extern "C" {
 	       PMIx layer error)					\
 	/** IV callback - cannot handle locally */			\
 	ACTION(DER_IVCB_FORWARD,	(DER_ERR_GURT_BASE + 24),	\
-	       IV callback - cannot handle locally)			\
+	       Incast variable)						\
```

Comment:
oh, sorry, that should have been s/IV/Incast variable.   But I guess what it is saying is really

Incast variable unavailable locally, must forward